CMakeLists.txt

```diff
@@ -79,6 +95,7 @@ if ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
     link_directories (${GSTREAMER_APP_LIBRARY_DIRS})
 endif()
 
-add_executable(camera_emulation_server "src/server.cpp")
+find_sources("src" "include")
+add_executable (camera_emulation_server ${HEADERS} ${SOURCES})
```

Comment:
Don't you want to rename everything to ces ?
Here you're having both names which is not very consistent.